Gemfile

```diff
@@ -21,6 +21,7 @@ gem 'puma'
 gem 'pg'
 gem 'omniauth-cas'
 gem 'rubocop'
+gem 'parser', '2.3.0.7'
```

Comment:
It's a dependency of something else which includes an older version. Old version or parser warned it might not be compatible with Ruby 2.3.0, and suggested upgrading.